frontend/src/components/cards/Issue.vue

```diff
@@ -1,14 +1,39 @@
 <template>
```

Comment:
Can we rename this file to fix the typo: "Issuer.vue"

Reply:
renamed